crypto/store/loader_file.c

```diff
@@ -748,7 +755,7 @@ static OSSL_STORE_LOADER_CTX *file_open(const OSSL_STORE_LOADER *loader,
         }
     } else {
         BIO *buff = NULL;
-        char peekbuf[4096];
+        char peekbuf[64];
```

Comment:
A .pem file may have arbitrary text before the starting line (I've seen numerous certs that contain the full output of `openssl x509 -in whatever -inform der -text`... that includes the PEM data *at the end*).

My 4k choice comes [from here](https://github.com/openssl/openssl/blob/master/crypto/bio/bf_buff.c#L23), because that's the maximum amount of characters BIO_f_buffer() can buffer.